### DISCUSSION TITLE
Possibilitty to dissable DDEV plugin for certain projects

### DISCUSSION THREAD

**Discussion by zhilinskiy** [php-perfect/ddev-intellij-plugin#148] [Ideas]
You've made a great plugin that really speeds things up. But I would like to be able to turn it off for some projects that do not use DDEV. Otherwise the IDE constantly gives out errors of the plugin. Thank you.

**Comment by nico-loeber** [comment]
Hey @zhilinskiy,

What kind of errors do you receive?
The idea of the plugin is to detect wether this is a ddev plugin and to only enable itself if ddev is installed and there is a .ddev directory.
A "disable for this project" setting can easily be added but I would like to avoid it as it might be annoying to eanble/disable it manually.
